src/main/java/gui/controller/ContentStatisticsController.java

```diff
@@ -16,20 +29,70 @@
 public class ContentStatisticsController implements Initializable {
 
     @FXML
-    private PieChart chart1;
-    
+    private PieChart chartMovies;
+    @FXML
+    private VBox vboxForLabels;
+
     @Override
     public void initialize(URL url, ResourceBundle rb) {
+
+        DBFacade db = new DBFacade();

```

Comment:
should calls into the DBFacade not at least be wrapped into some sort of application controller like it is done with DataHandler?